pages/infolettres.html

```diff
@@ -0,0 +1,52 @@
+---
+layout: newsletters
+title: Lettres d'informations
+permalink: /infolettres/
+class: newsletters
+---
+
+<div class="tpl-newsletters__content">
+  <div class="mailjet">
+    <iframe
+      class="mj-w-res-iframe"
+      frameborder="0"
+      scrolling="no"
+      marginheight="0"
+      marginwidth="0"
+      src="https://app.mailjet.com/widget/iframe/zR1/IjQ">
+    </iframe>
+    <script type=text/javascript"" src="https://app.mailjet.com/statics/js/iframeResizer.min.js"></script>
```

Comment:
Je crois qu'il y a une typo

```suggestion
    <script type="text/javascript" src="https://app.mailjet.com/statics/js/iframeResizer.min.js"></script>
```

Reply:
En effet 😅